lib/cassia/response_handlers/close_scan.rb

```diff
@@ -0,0 +1,32 @@
+module Cassia
+  module ResponseHandlers
+    class CloseScan
+      def initialize(access_controller, aps: )
+        @access_controller = access_controller
+        @aps = aps
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success
+        routers_to_be_turned_on = @access_controller.routers.select {|router| @aps.include?(router.mac) }
```

Comment:
I think we should rename this variable to `routers_to_open_scan` or something along those lines. Also we should handle the case of `*` here too